app-backend/tile/src/main/scala/LayerCache.scala

```diff
@@ -53,7 +52,14 @@ object LayerCache extends Config {
 
   def attributeStore(bucket: String, prefix: String): Future[S3AttributeStore] =
     caching[S3AttributeStore, InMemoryRepr](s"store-$bucket-$prefix"){
```

Comment:
I didn't realize we can use the `("histogram", id, zoom)`-like syntax for the cache key, that seems better, we should just use that consistently.